### ISSUE TITLE
Plugin removal

### ISSUE THREAD

**Issue by tprelog** [tprelog/iocage-homeassistant#62]
In light of recent changes to Home Assistant, I’ve decided to have this removed from the community plugins on TrueNAS CORE. As Home Assistant continues to grow and add new features, installation on FreeBSD no longer “just works” as it had for so many years in the past. I will leave this artifact repo on GitHub for those who wish to continue using the plugin or add the files to a regular jail. However, I can not leave this so easily accessible when Home Assistant’s own default configuration no longer works as expected.

Existing "plugin" installations will continue to work and can be treated as a "regular" jail.

**Comment by tprelog** [comment]
Is it worth the effort of converting this back to "regular jail" installation script?

If enough people are still planning to use this, I don't mind doing the work. I just don't want to waste my time for nothing.

**Comment by adorobis** [comment]
> Is it worth the effort of converting this back to "regular jail" installation script?
> 
> If enough people are still planning to use this, I don't mind doing the work. I just don't want to waste my time for nothing.

Personally I would direct the efforts to migration to a different platform. Either SCALE with docker or something completely different. The FreeBSD is a dead end for HA so I don't think it makes sense to invest any more time in it. Your experience with migration and running HA on SCALE/docker would be very useful I'm sure :)

**Comment by adorobis** [comment]
> considering buying the HA Yellow

I have just moved to a mini PC, running on Celeron N3060 (2 core), 8GB RAM and an 128GB SSD (smallest I've had laying around). Seems it is perfectly ok for HA and a few other components (e.g. esphome, appdaemon, tasmoadmin etc). It uses 5-7W of power on average and can be bought for less than 100USD/EUR. 
But for now I've left the HAs MariaDB on TrueNAS as I'm not sure if the hardware would be good enough (plus it was so easy to just move the HA) as well as a few other things (zigbee2mqtt, zwavejs2mqtt, etc). No pressure to migrate to Scale anymore and I'm running HA on one of the 2 mostly recommended installation methods: Docker container on Debian - running headless, without any desktop GUI components.

**Comment by elyobelyob** [comment]
Ideally I want to run Nginx, MQTT and HA on a single low powered machine. Something like what you suggest may be a better choice? My Zigbee experience has been a bit hit and miss on FreeBSD as using an external Sonoff coordinator.

Any decent machines to look at that will run the bluetooth and zigbee stuff? Based in UK, and will start investigating. 

Reducing my 60W FreeBSD machine usage would also be a huge win.

TIA.

**Comment by jkhii** [comment]
I may be in the minority but I would certainly use a jail installation
script. Either as is, or with some local modifications to suit my odd
network needs.

What I currently have is very old but based on your plugin from a few years
ago.

Thanks for all the time you have put into this.

Kelly

**Comment by adorobis** [comment]
> Reducing my 60W FreeBSD machine usage would also be a huge win.

Not sure if you'll be able to reduce much, depending on your setup it might be the hdds that use the most of power.

> My Zigbee experience has been a bit hit and miss on FreeBSD as using an external Sonoff coordinator

In my case there are two main problems: in FreeBSD you can't (easily) refer to usb device by name so the device name is changing e.g. after reboot. You can resolve it by a piece of scripting that is executed on the jail start. 
The second problem is that occasionally my zigbee coordinator (conbee 2) is getting disconnected and I have to unplug it and plug back in. Not sure if this is freebsd related though.

**Comment by aliekiddo** [comment]
I had a go at migrating to scale because of this and it was a disaster. For whatever reason all of the apps I tried to install got stuck in 'deploying' so I'm back on Core for now.

I would love to be able to continue to run HA on TrueNAS Core. I've been using FreeNAS / TrueNAS for nearly 10 years from several old PC's; I don't have the ability / room to run a proper server and it's been a great way for myself as a home user to run a server with what I already have.

The only reason I ever tried Home Assistant is because it could be run on my TrueNAS server.

Agreed with @jkhii Thanks for all your hard work @tprelog

**Comment by adorobis** [comment]
> The only reason I ever tried Home Assistant is because it could be run on my TrueNAS server.

That was my induction to HA as well. But as we know it running it longer on a not supported platform is a risk of running headlong to the wall. I would highly recommend to move to a supported way off running HA rather sooner than later

**Comment by adorobis** [comment]
> The best way to run containers on scale is to use docker compose, unfortunately they are killing off that approach officially.

Earlier this year they have released docker-compose app in the TrueCharts store and it works pretty well. Anyway I've also moved to a pure debian running HA via docker-compose. You can also add portainer to add gui management for all containers if you like it over command line

**Comment by tprelog** [comment]
> I presume, unless I have a catastrophic failure in the near future, I won't be needing a regular jail?

Correct. It will also remain possible to (re)install the "plugin" using the command line, even though it's not recommended.
